### DISCUSSION TITLE
Android Payment Question

### DISCUSSION THREAD

**Discussion by Westy2049** [chuot/rdio-scanner#209] [General]
It appears that I now have to sign up for a premium account to download?  This only happened after I uninstalled and reinstalled the app.  Does the $5 per month I pay not allow me to download any more?

Thanks

Dan

**Comment by Westy2049** [comment]
Chrystian - Can you please answer?

**Comment by chuot** [comment]
Just to make sure I understand you say you bought a premium subscription from the native app and then uninstalled the app for whatever reason and now you can no longer download the app from Google Play ? Can you post a screenshot of the message you have on screen? I'll look at it tomorrow when I get to the office. Nothing has changed since the last release 1.6.0 on May 25th. I wonder what could it be?

**Comment by Westy2049** [comment]
-------- Original message --------From: Chrystian Huot ***@***.***> Date: 6/12/22  11:52  (GMT-05:00) To: chuot/rdio-scanner ***@***.***> Cc: Westy2049 ***@***.***>, Author ***@***.***> Subject: Re: [chuot/rdio-scanner] Android Payment Question (Discussion #209) 
Just to make sure I understand you say you bought a premium subscription from the native app and then uninstalled the app for whatever reason and now you can no longer download the app from Google Play ? Can you post a screenshot of the message you have on screen? I'll look at it tomorrow when I get to the office. Nothing has changed since the last release 1.6.0 on May 25th. I wonder what could it be?

**Comment by Westy2049** [comment]
When you opened it up to Chester county I went to your site and paid the monthly.  I'm thinking that I downloaded it from your site.  Got to thinking yesterday that I haven't checked for updates lately so I head over to your site and sees it says to download via play store so I did and uninstalled the previous version.  The play store has a different icon then the downloaded one. Anyways I downloaded and connected using the play store and get that message.Is there a direct download not using play store that I should be using?
-------- Original message --------From: Chrystian Huot ***@***.***> Date: 6/12/22  11:52  (GMT-05:00) To: chuot/rdio-scanner ***@***.***> Cc: Westy2049 ***@***.***>, Author ***@***.***> Subject: Re: [chuot/rdio-scanner] Android Payment Question (Discussion #209) 
Just to make sure I understand you say you bought a premium subscription from the native app and then uninstalled the app for whatever reason and now you can no longer download the app from Google Play ? Can you post a screenshot of the message you have on screen? I'll look at it tomorrow when I get to the office. Nothing has changed since the last release 1.6.0 on May 25th. I wonder what could it be?

**Comment by chuot** [comment]
There seems to be some confusion here. The official links for the Rdio Scanner open source project and native apps are listed here in the `README.md`, in the PDF document provided with the open source executable and finally on the `rdio-scanner.io` website. If the links you are referring to come from a site that uses our open source project and charges its users for access, then you should address the situation directly with them. Keep in mind that I am not affiliated with these sites and have no control over them. I'm just hoping that some of them will eventualy sponsor the open source project that I built from the very first line of code 3 years ago.

**Comment by Westy2049** [comment]
Attached
-------- Original message --------From: Chrystian Huot ***@***.***> Date: 6/12/22  11:52  (GMT-05:00) To: chuot/rdio-scanner ***@***.***> Cc: Westy2049 ***@***.***>, Author ***@***.***> Subject: Re: [chuot/rdio-scanner] Android Payment Question (Discussion #209) 
Just to make sure I understand you say you bought a premium subscription from the native app and then uninstalled the app for whatever reason and now you can no longer download the app from Google Play ? Can you post a screenshot of the message you have on screen? I'll look at it tomorrow when I get to the office. Nothing has changed since the last release 1.6.0 on May 25th. I wonder what could it be?
